packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.tsx

```diff
@@ -88,6 +90,25 @@ const BuySellModal = () => {
         }
     };
 
+    const onSubmit = () => {
+        if (has_rate_changed) {
+            showModal({
+                key: 'MarketRateChangeErrorModal',
+                props: {
+                    submitForm,
+                    values: {
+                        currency: buy_sell_store.account_currency,
```

Comment:
can we not directly access the props from the modal?

Reply:
we dont pass any props to this modal, they are stored inside of the buy_sell_store in v1, unlike in v2 where we do have all the props passed